transform-plugins/docs/XMLParser-transform.md

```diff
@@ -0,0 +1,82 @@
+# XML Parser Transform
+
+Description
+-----------
+The XML Parser uses XPath to extract field from a complex XML Event. This should generally be used in conjunction with
+the XML Reader Batch Source. The XML Source Reader will provide individual events to the XML Parser and the XML Parser
+will be responsible for extracting fields from the events and mapping them to output schema.
+
+
+Use Case
+--------
+The transform takes input record that contain xml events/records, parses it using the xpath specified and returns a
+structured record according to the schema specified by user.
+For example, this plugin can be used in conjuction with XML Reader Batch Source to extract values from XMLNews documents
+and to create structured records which will be easier to query.
+
+
+Properties
+----------
+
+**input:** Specifies the field in input that should be considered as source of XML event or record.
+
+**encoding:** Specifies XML encoding type(default is UTF-8).
+
+**xpathMappings:** Specifies a mapping from XPath of XML record to field name.
```

Comment:
Mapping of the field names to the XPath of the XML record. A comma-separated list, each element of which is a field name, followed by a colon, followed by the XPath location path.